tests/test_cases/pipeinput2.c

```diff
@@ -31,7 +31,7 @@ int main(int argc, char *argv[])
         close(pipefd[0]);          /* Close unused read end */
         printf("about to write to pipe in parent\n");
         printf("I'm parent with pid: %d\n", cpid);
-
+    fflush(stdout);
```

Comment:
This might be a bit charry picking, but would it be possible to fix the indentation?